sql/moz-fx-data-shared-prod/marketing_campaign_names/asa_v1/metadata.yaml

```diff
@@ -0,0 +1,2 @@
+description: Extract fields from ASA v1 campaign name
```

Comment:
Generally light on descriptions because these won't be used directly by end users